### PR TITLE
chore: Rename all .story.* files to .stories.*

### PR DESCRIPTION
**Pull request by lukelooker**
File rename accomplished with:
`find . -name '*.story.tsx' -exec bash -c ' mv $0 ${0/\story/stories}' {} \;`

Then bulk find & replace for `.story` import and minor config file tweaks.

### CODE REVIEW

File: storybook/.storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<style>
+  .sb-show-main, .sbdocs-wrapper {
+    padding: 0 !important; /* stylelint-disable-line declaration-no-important */
+  }
+  .sbdocs-title {
+    display: none;
+  }
+</style>
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;500;600;700&family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
```
Comment (ElliotTheHuman) [inline storybook/.storybook/preview-head.html:10]: Dumb question, but does this file get "loaded" in anywhere / mentioned in a JS file? Or is there just storybook magic that automatically picks up that we have a `preview-head.html` and loads in the relevant fonts / additional styles?

**Review by ElliotTheHuman — APPROVED** [review]
LGTM, but seeing a conflict. Happy to re-thumbs up once that's resolved.

File: storybook/.storybook/preview-head.html (hunk 1 above)
Comment (lukelooker) [reply storybook/.storybook/preview-head.html:10]: Just "magic" - https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head

**Review by ElliotTheHuman — APPROVED** [review]
🎉 🎉 🎉

**Pull request merged**